store_monitoring/crud.py

```diff
@@ -16,14 +18,22 @@ def insert_store_status(db):
             db.refresh(data)
     return
 
+def string_to_time(time):
```

Comment:
1) move this to a separate file. Create a utils folder and inside it, create a time file.
2) rename it, it doesn't convert all time strings to time, right? It can only do it for business hours. So name it accordingly